coordinates/w64h32.json.example

```diff
@@ -126,14 +126,18 @@
       "width": 24
     },
     "pitch": {
-      "font_name": "4x6",
       "x": 1,
       "y": 50,
-      "width": 41,
       "enabled": false,
       "mph": false,
       "desc_length": "Short"
     },
+    "pitch_count": {
+      "x": 1,
```

Comment:
This one looks like it's missing `font_name`

Reply:
Yeah, I deleted it there since 4x6 is the default for this size. It’s not clear to me if we even should be setting it on the other boards in the examples